@here/harp-geoutils/lib/coordinates/GeoPolygon.ts

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright (C) 2017-2020 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { Vector2 } from "three";
+
+import { GeoBox } from "./GeoBox";
+import { GeoCoordinates, MAX_LATITUDE, MIN_LATITUDE } from "./GeoCoordinates";
+import { GeoCoordinatesLike } from "./GeoCoordinatesLike";
+import { GeoPolygonLike } from "./GeoPolygonLike";
+
+type MinThreeItemsArray<T> = [T, T, T, ...T[]];
+
+export type GeoPolygonCoordinates = MinThreeItemsArray<GeoCoordinatesLike | GeoCoordinates>;
```

Comment:
`GeoCoordLike` would make it even more flexible for users

Reply:
yes, I left it out, as it has a different interface and therefore convolutes the code, but let me check how..

Reply:
done